Sources/OpenAPIRuntime/URICoder/Serialization/URISerializer.swift

```diff
@@ -228,8 +232,15 @@ extension URISerializer {
         case (.simple, false):
             keyAndValueSeparator = ","
             pairSeparator = ","
+        case (.deepObject, _):
```

Comment:
I think we should again validate that explode is true, and throw an error on false.